approved-book-list.json

```diff
@@ -1706,6 +1713,11 @@
       "collection_id": "col10280",
       "content_version": "1.3.5",
       "min_code_version": "99999999.999999"
+    },
+    {
+      "collection_id": "col10246",
+      "content_version": "1.3.2",
+      "min_code_version": "20210623.195337"
```

Comment:
@Jacqueline-Yarbrough - I suspect this new entry is so the book will build in CORGI, is that right? If so, I think we want to use a `min_code_version` of `"99999999.999999"` so that the pipeline ignores it. Once it's on production cnx, we can replace with a valid minimum code version.

@ne-newton - Is there a good place to document this convention we made up yesterday 😂 ?

Reply:
ok i will make that change, apologies i didn't see this until after i merged

Reply:
No worries!